### DISCUSSION TITLE
Is this mod compatible with Fresh Animations?

### DISCUSSION THREAD

**Discussion by FireLynxx** [MrKacafirekCZ/doggo-mod-overhauled#38] [Q&A] [answered]
Simple question asking if this is compatible with Fresh Animations, I want the other animations and the ones in this so I'm simply asking if they can work together :D

**Comment by MrKacafirekCZ** [comment, marked as answer]
Hello, unfortunately as far as I'm aware Fresh Animations aren't compatible with this mod for multiple reasons:
1) Doggo Mod Overhauled adds a new entity which only looks like the vanilla wolf but in reality it isn't the vanilla wolf therefore Fresh Animations have no effect.
2) Fresh Animations require OptiFine to run and if I remember correctly Doggo Mod Overhauled had some issues with OptiFine alone.

**Reply by SylisMC** [reply]
im running fresh animations on fabric fine with doggo. the dog isnt affected but everything else (including wolfes before they become doggos work fine)

**Reply by MrKacafirekCZ** [reply]
Hm, interesting. Maybe I will have to look at this one again.

**Comment by Some-Doofus** [comment]
Yep you can run fresh animations without optifine as long as you have [Entity Model Features](https://modrinth.com/mod/entity-model-features) installed. Maybe you could collab with the creator of FA to make a patch for this mod?

**Reply by MrKacafirekCZ** [reply]
Interesting but point 1 still stands. According to the author's FAQ page, they say:

> Q: Why isn't the ___ animated? Can you add it?
> A: Not all the mobs are animated yet. Fresh Animations is a Beta pack so it is developed over time, so new mobs animations will be added in major updates. I work and add new animations at my own pace, so please refrain from requesting animations.

To me this sounds like they aren't even done adding all vanilla mobs and likely have not even considered adding modded mobs.

**Comment by BigHatKC** [comment]
This mod is **_cripplingly_** unfinished and poorly structured, meaning if you want ANY mods for tameable pets or specifically wolves, you're EXCLUSIVELY stuck to this one. Just stick with the vanilla wolves, since every dumbass mod-author with their head up their ass wants to make a new entity instead of actually modifying the existing ones--and WON'T tell you about it.
